controllers/ibmblockcsi_controller.go

```diff
@@ -308,6 +317,14 @@ func (r *IBMBlockCSIReconciler) updateStatus(instance *ibmblockcsi.IBMBlockCSI,
 		return err
 	}
 
+	callHomeCronJob, err := r.getCallHomeCronJob(instance)
+	if err == nil {
+		err = r.updateCallHomeStatus(instance, callHomeCronJob, logger)
```

Comment:
this doesn't seem to update any status

Reply:
how would you call it?

Reply:
since it's only used one, I would move its `if` over here, and call a shorter `deleteCallHomeCronJob` function

Reply:
done